R/package.r

```diff
@@ -124,7 +124,7 @@ vignette_index <- function(path = ".") {
 
   title <- file.path(path, "vignettes", vig_path) %>%
     purrr::map(rmarkdown::yaml_front_matter) %>%
-    purrr::map_chr("title", .null = "UNKNOWN TITLE")
+    purrr::map_chr("title", .default = "UNKNOWN TITLE")
```

Comment:
If you're going to make this change, you need to also update the DESCRIPTION to require the dev version of purrr (in imports) and automatically install it with `install_github` (by adding to the remotes field)